docker_config/compose_files/docker-compose.yml

```diff
@@ -337,7 +337,7 @@ services:
       --cluster-host ${CODALAB_WORKER_MANAGER_GPU_KUBERNETES_CLUSTER_HOST}
       --auth-token ${CODALAB_WORKER_MANAGER_GPU_KUBERNETES_AUTH_TOKEN}
       --cert-path ${CODALAB_WORKER_MANAGER_GPU_KUBERNETES_CERT_PATH}
-      --cert ${CODALAB_WORKER_MANAGER_GPU_KUBERNETES_CERT}
+      --cert "${CODALAB_WORKER_MANAGER_GPU_KUBERNETES_CERT}"
```

Comment:
Read this directly as an env variable CODALAB_WORKER_MANAGER_GPU_KUBERNETES_CERT